src/ophyd_async/core/device_save_loader.py

```diff
@@ -152,9 +147,6 @@ def save_to_yaml(phases: Sequence[Dict[str, Any]], save_path: str) -> None:
 
     yaml.add_representer(np.ndarray, ndarray_representer, Dumper=yaml.Dumper)
 
-    yaml.add_representer(ca_float, ca_dbr_representer, Dumper=yaml.Dumper)
-    yaml.add_representer(ca_int, ca_dbr_representer, Dumper=yaml.Dumper)
-    yaml.add_representer(ca_str, ca_dbr_representer, Dumper=yaml.Dumper)
     yaml.add_representer(ca_array, ca_dbr_representer, Dumper=yaml.Dumper)
```

Comment:
Should: I think we can just change this to 
```suggestion
    yaml.add_representer(ca_array, ndarray_representer, Dumper=yaml.Dumper)
```

and remove `ca_dbr_representer`